tests/keras/layers/wrappers_test.py

```diff
@@ -109,6 +109,43 @@ def test_TimeDistributed():
     assert uid in td._input_map
     assert K.int_shape(td._input_map[uid]) == (None, 2)
 
+    # test layers with multiple outputs
+
+    # define layer
+    def func(x):
+        return [x * 0.2, x * 0.3]
+
+    def output_shape(input_shape):
+        return [input_shape, input_shape]
+
+    def mask(inputs, mask=None):
+        return [None, None]
+
+    # test single layer with multiple outputs
+    x = Input(shape=(3, 2))
+    x1 = Lambda(function=func,
+                output_shape=output_shape,
+                mask=mask)(x)
+
+    model = Model(inputs=x, outputs=x1)
+    model.compile(optimizer='rmsprop', loss='mse')
+    model.fit(x=np.random.random((16, 3, 2)),
```

Comment:
Will do, thanks, should I replace the existing ones as well or should I leave that for a different PR?